server/storage/sql_models.go

```diff
@@ -11,6 +11,16 @@ const (
 	changeCategoryDeletion = "deletion"
 )
 
+// TUFFileTableName returns the name used for the tuf file table
+var TUFFileTableName = func() string {
```

Comment:
Didn't mean to let this stagnate, I've been trying to evaluate which of these changes are really needed.

I was thinking that the functions might help for threshold "witnessing" (or whatever the term is now) by creating a different table for "staged" metadata. But since that design is still very up in the air I will change this and update the PR.